datamade/committee-oversight
# Hearing detail and list pages

## Overview

Closes #110. This PR creates templates for hearing detail pages and a Wagtail hearing list page. It reorganizes the hearing editing/creating/deleting pages to accommodate this phase of the project, and creates links between the Committee, Hearing, and Category pages wherever possible.

This PR also updates the README to reflect changes to the dbload service made in #111.

**Note:** All hearings should now have detail pages, but Lugar will create Committee and Category detail pages manually as they enter their descriptions of both. The CMS fixture in this PR includes a sample page of each: the [House Committee on Science, Space, and Technology](http://localhost:8000/committee-2c6395b2-dc9d-45f6-a4db-481b1b750431/) and the category page for [Nominations](http://localhost:8000/category-nominations/)

## Testing Instructions

 * Follow the instructions in the [README from this PR](https://github.com/datamade/committee-oversight/tree/issue-110-hearing-details) to get the site running
* Open http://localhost:8000/ and click `Browse Hearings` in the navbar. Make sure you see a list of the categories and a table of hearings
* Click `Nominations` in the category list and make sure the link works
* Go back to the `Browse Hearings` page and click on a particular hearing. Does it look as you'd expect?
* Log into the site and click around. Can you edit and create pages as you'd expect? Are the pages linked as you'd expect them to be?
